src/modules/settings/components/clients-selector.tsx

```
@@ -0,0 +1,78 @@
+          {kClients.map((client, index) => (
+            <li
+              className={cn("rounded-lg transition-all p-4 cursor-pointer", {
+                "shadow-lg": kClientId !== client.id,
+                shadow: kClientId === client.id,
+              })}
+              key={index + client.id}
+              onClick={() => setKClientId(client.id)}
+            >
+              <p className="font-bold text-sm">{client.name}</p>
+              {client.kVats.map(({ vat }) => (
+                <p className="font-mono text-sm">{vat}</p>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)